examples/500-cli/cli.md

```diff
@@ -421,92 +418,45 @@ warpforge catalog ls
 
 [testmark]:# (catalog/then-add-tar/sequence)
 ```
-warpforge catalog --name my-catalog add tar alpinelinux.org/alpine:v3.15.0:x86_64 https://dl-cdn.alpinelinux.org/alpine/v3.15/releases/x86_64/alpine-minirootfs-3.15.0-x86_64.tar.gz
+warpforge catalog --name my-catalog add tar warpsys.org/busybox:v1.35.0:amd64-static file://.warpforge/warehouse/4z9/DCT/4z9DCTxoKkStqXQRwtf9nimpfQQ36dbndDsAPCQgECfbXt3edanUrsVKCjE9TkX2v9
 ```
 
 [testmark]:# (catalog/then-add-tar/then-check/script)
 ```
-cat .warpforge/catalogs/my-catalog/alpinelinux.org/alpine/_module.json
-cat .warpforge/catalogs/my-catalog/alpinelinux.org/alpine/_releases/v3.15.0.json
-cat .warpforge/catalogs/my-catalog/alpinelinux.org/alpine/_mirrors.json
+cat .warpforge/catalogs/my-catalog/warpsys.org/busybox/_module.json
+cat .warpforge/catalogs/my-catalog/warpsys.org/busybox/_releases/v1.35.0.json
+cat .warpforge/catalogs/my-catalog/warpsys.org/busybox/_mirrors.json
 ```
 
 [testmark]:# (catalog/then-add-tar/then-check/output)
 ```
 {
 	"catalogmodule.v1": {
-		"name": "alpinelinux.org/alpine",
+		"name": "warpsys.org/busybox",
 		"releases": {
-			"v3.15.0": "zM5K3Skrij8UUcQM7jGuwk35okUEg3qpc8CT38zrdQP7xTesb6we5VThECimvJMt5qwd6gc"
+			"v1.35.0": "zM5K3Z62CY9X6QkccuptyiC3a1tC32Fh2n1ujF8KH5Fz1BvKqppWJZgQJxEgypvF3pqzhdE"
 		},
 		"metadata": {}
 	}
 }
 {
-	"releaseName": "v3.15.0",
+	"releaseName": "v1.35.0",
 	"items": {
-		"x86_64": "tar:57j2Ee9HEtDxRLE6uHA1xvmNB2LgqL3HeT5pCXr7EcXkjcoYiGHSBkFyKqQuHFyGPN"
+		"amd64-static": "tar:4z9DCTxoKkStqXQRwtf9nimpfQQ36dbndDsAPCQgECfbXt3edanUrsVKCjE9TkX2v9"
 	},
 	"metadata": {}
 }
 {
 	"catalogmirrors.v1": {
 		"byWare": {
-			"tar:57j2Ee9HEtDxRLE6uHA1xvmNB2LgqL3HeT5pCXr7EcXkjcoYiGHSBkFyKqQuHFyGPN": [
-				"https://dl-cdn.alpinelinux.org/alpine/v3.15/releases/x86_64/alpine-minirootfs-3.15.0-x86_64.tar.gz"
+			"tar:4z9DCTxoKkStqXQRwtf9nimpfQQ36dbndDsAPCQgECfbXt3edanUrsVKCjE9TkX2v9": [
+				"file://.warpforge/warehouse/4z9/DCT/4z9DCTxoKkStqXQRwtf9nimpfQQ36dbndDsAPCQgECfbXt3edanUrsVKCjE9TkX2v9"
 			]
 		}
 	}
 }
 ```
 
-#### git
```

Comment:
Maybe let's keep these, but just figure out how to put a feature flag on these tests?

I don't know what the gopherean state of the art is for that... there's the `t.Short` predicate, but I'm not a huge fan of that because it's just one bool.  Env vars connected to a `t.Skip` would be an option.  Build tags are maybe also an option?  I don't have strong opinions.

Reply:
For now, I have put them back in and removed the testmark line... so the example is still there but there but it will not run. Looking for a better solution to allow net/no-net options.

Reply:
Now behind the `-testutil.offline` flag.